integration_tests/mockApis/courseParticipations.ts

```diff
@@ -0,0 +1,23 @@
+import type { SuperAgentRequest } from 'superagent'
+
+import { apiPaths } from '../../server/paths'
+import { stubFor } from '../../wiremock'
+import type { CourseParticipation, Person } from '@accredited-programmes/models'
+
+export default {
+  stubCreateParticipation: (args: {
+    courseParticipation: CourseParticipation
+    prisonNumber: Person['prisonNumber']
```

Comment:
The `prisonNumber` isn't user here (and could be taken from the course participation if needed)

Reply:
Good spot, intended to use it but then turned out I didn't need it.